src/components/sequencing/Sequences.svelte

```diff
@@ -59,6 +63,41 @@
     const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
     goto(`${base}/sequencing/new${workspaceId ? `?${SearchParameters.WORKSPACE_ID}=${workspaceId}` : ''}`);
   }
+
+  async function importLibarySequences(): Promise<void> {
+    const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
+    if (workspaceId === null) {
+      console.log("Workspace doesn't exist");
+      showFailureToast("Library Import: Workspace doesn't exist");
+      return;
+    }
+    const { confirm, value } = await showLibrarySequenceModel();
+
+    if (confirm && value) {
+      const fileName = await value.libraryFile.name;
+      const type = fileName.slice(fileName.lastIndexOf('.') + 1);
+      if (type !== 'satf') {
+        console.log(`Unsupported file type ${type}`);
+        showFailureToast('Library Import: Unsupported file type');
+        return;
+      }
+      const contents = await value.libraryFile.text();
```

Comment:
I think you might need to add a try/catch for if it's unable to read the file.

Reply:
Got it I added a try-catch